extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -161,8 +161,22 @@ export class AmpAnalytics extends AMP.BaseElement {
           if (!result) {
             return;
           }
-          addListener(this.getWin(), trigger,
-              this.handleEvent_.bind(this, trigger));
+
+          if (trigger['selector']) {
+            // Expand the selector using variable expansion.
+            trigger['selector'] = this.expandTemplate_(trigger['selector'],
+                trigger);
+            return urlReplacementsFor(this.getWin()).expand(trigger['selector'])
```

Comment:
One think it's a good first step. We'll just have to make a note in docs that only author-based vars are allowed here.